tools/SetupFlow/DevHome.SetupFlow/ViewModels/QuickstartPlaygroundViewModel.cs

```diff
@@ -611,6 +679,23 @@ public void OnQuickstartSelectionChanged()
         }
     }
 
+    private void SetupChat()
+    {
+        ChatMessages.Clear();
+
+        ChatMessages.Add(new ChatStyleMessage
+        {
+            Name = "Hello! What kind of project would you like to create? Try a sample prompt or write one of your own.",
+            Type = ChatStyleMessage.ChatMessageItemType.Response,
+        });
+
+        ChatMessages.Add(new ChatStyleMessage
+        {
+            Name = "If you would like to help us guide you to a good prompt, hit the Enter/Return key on your keyboard or press Submit after typing in your prompt. Otherwise, simply click on Generate to ignore prompt guidance.",

```

Comment:
Localization.   

I don't know if "simply" can be used.  Some people might not find clicking on "generate" to be a simple task.

Reply:
NIT: "ignore" is a bit harsh.  `Otherwise, click on Generate` seems good enough.